test/parallel/test_tensorflow.py

```diff
@@ -4051,6 +4051,8 @@ def test_horovod_join_allreduce(self):
                 self.assertSequenceEqual(ret_values, [ret] * size,
                                          msg="hvd.join() did not return the same value on each rank")
 
+    @pytest.mark.skipif(LooseVersion(tf.__version__) >=
```

Comment:
Can you create an issue for this and link here, please?

Reply:
added.